fix: Updating the School Timetabling quickstarts documentation (#494)
TimefoldAI/timefold-solver: docs/src/modules/ROOT/pages/quickstart/spring-boot/spring-boot-quickstart.adoc

        "timefold.solver.termination.best-score-limit=0hard/*soft"})
public class TimetableControllerTest {

    @Autowired
    private TimetableController timetableController;

    @Test
    @Timeout(600_000)
    public void solveDemoDataUntilFeasible() throws InterruptedException {
        timetableController.solve();
        Timetable timetable = timetableController.getTimetable();
        while (timetable.getSolverStatus() != SolverStatus.NOT_SOLVING) {
            // Quick polling (not a Test Thread Sleep anti-pattern)
            // Test is still fast on fast machines and doesn't randomly fail on slow machines.
            Thread.sleep(20L);
            timetable = timetableController.getTimetable();
        }
        assertFalse(timetable.getLessonList().isEmpty());
        for (Lesson lesson : timetable.getLessonList()) {
            assertNotNull(lesson.getTimeslot());
            assertNotNull(lesson.getRoom());
        }
        assertTrue(timetable.getScore().isFeasible());
    }

}
----

. Build an attractive web UI on top of these REST methods to visualize the timetable.

> For an implementation with UI-integrated and in-memory storage, check out {spring-boot-quickstart-url}[the Spring-boot quickstart source code].

Reviewer: ```suggestion
For a full implementation with a web UI and in-memory storage, check out {spring-boot-quickstart-url}[the Spring-boot quickstart source code].
```
